go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/grafana/grafana-operator/v5
 
-go 1.20
+go 1.21
+
+toolchain go1.21.3
```

Comment:
Introduced by a `go mod tidy` required by the makefile targets. Can exclude this from the PR